agimus_controller/agimus_controller/ocp/ocp_croco_generic.py

```diff
@@ -167,6 +173,72 @@ def build(self, data: BuildData):
         return crocoddyl.ResidualModelFramePlacement(data.state, id, pref)
 
 
+@dataclasses.dataclass
+class ResidualModelVisualServoing(ResidualModel):
+    """Visual servoing calculated as:
+    wMf_target = wMo_vision * oMf_target
+    where:
+    - wMf_target is the target pose of the frame in the world frame
+    - wMo_vision is the pose of an object frame in the world frame
+    - oMf_target is the target pose of the frame in the object frame
```

Comment:
I'm not sure what's the best naming, I would see something like
```suggestion
    - wMee_target is the target pose of the end-effector frame in the world frame
    - wMo_vision is the pose of an object frame in the world frame
    - oMee_target is the target pose of the end-effector frame in the object frame
```

Reply:
I have updated the doc.